etc/oracle-admin/partitions.sql

```diff
@@ -0,0 +1,495 @@
+/*
+Objective:
+==========
+	The following PL/SQL code will be used to create 
+	an automatic process to remove 90 days older partitions 
+	in FILETRANSFERSDB and FILEMETADATA tables in production database.
+
+Database: 
+=========
+    These code is applied to the following databases:
+
+	- production:    cms_analysis_reqmgr@cmsr
+    - preproduction: cmsweb_analysis_preprod@int2r
+
+Dependency/Requirements:
+========================
+	1. We must ensure that we have proper permission to create 
+	scheduler jobs and send email alerts. If we do not have it
+	then ask from Oracle support.
+
+*/
+
+/*
+--------------------------------------------------------------------------------------------------------------------------
+
+*/
+
+set echo on;
+
+/*
+
+Since oracle 23.c (which is not in production yet at CERN IT), this can be replaced by 
+Ref: https://stackoverflow.com/a/15633220
+
+CREATE OR REPLACE PROCEDURE CREATE_TABLE_PTNAME
+IS
+BEGIN
+
+	create table PTNAME (pname varchar2(100),tname varchar2(100));
+
+END CREATE_TABLE_PTNAME;
+
+
+*/
+
+
+CREATE OR REPLACE PROCEDURE CREATE_TABLE_PTNAME
+IS
+c NUMBER;
+v_sql clob;
+BEGIN
+        select count(*) into c  from user_tables   where table_name='PTNAME';
+        IF c=0 THEN
+				v_sql := '
+				create table PTNAME
+				(
+					pname varchar2(100),
+					tname varchar2(100)
+				)';
+
+				execute immediate v_sql;
+        END IF;
+
+END CREATE_TABLE_PTNAME;
+/
+
+/*=========================================================================================================================*/
+
+CREATE OR REPLACE FUNCTION  TIME_TAKEN
+
+	( start_time TIMESTAMP,end_time   TIMESTAMP )
+	return varchar2
+IS
+	data varchar2(500);
+BEGIN
+	data     := 	    '<b>Above Query execution time</b>'||
+                            '<table border=2><tr><td>Start Time</td><td>'|| TO_CHAR(start_time)||'</td></tr>'||
+                            '<tr><td>End Time</td><td>' || TO_CHAR(end_time) || '</td></tr>'||
+                            '<tr><td>Diff Time</td><td>' || TO_CHAR(end_time-start_time)||'</td></tr></table>';
+	return data;
+
+END TIME_TAKEN;
+/
+
+
+/*=========================================================================================================================*/
+CREATE OR REPLACE PROCEDURE SEND_EMAIL
+( data  IN varchar2 )
+IS
+	username varchar2(100);
+	dbname   varchar2(100);
+	emails   varchar(200) := 'cms-service-crab-operators@cern.ch';
+	  /*emails   varchar(200) := 'alccs.prajesh.sharma@gmail.com,prajesh.sharma@cern.ch,Stefano.Belforte@cern.ch,daina.dirmaite@cern.ch';*/
```

Comment:
do we need to keep those lines with old emails ?

Reply:
nope